muspinsim_plot/muspinsim_plot.xml

```diff
@@ -0,0 +1,105 @@
+<tool id="muspinsim_plot" name="Plot Muspinsim Results" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="3.5.1">matplotlib</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       python ${__tool_directory__}/generate_plot.py inputs.json
+    ]]></command>
+    <configfiles>
+        <inputs name="inputs" data_style="paths" filename="inputs.json" />
+    </configfiles>
+    <inputs>
+        <param type="text" name="title" label="Plot Title" value="" />
+        <param type="text" name="xlab" label="Label for x axis" value="" />
+        <param type="text" name="ylab" label="Label for y axis" value="" />
+        <param name="outfile_type" type="select" label="Output File Type" >
+            <option value="pdf">PDF</option>
+            <option value="png">PNG</option>
+        </param>
+        <repeat name="mu_out_series" title="Muspinsim File Series">
+            <param type="data" name="mu_data" label="Muspinsim Experiment Data (.dat)" />
+            <param type="text" name="mu_label" optional="true" label="Muspin Experiment Label" value="" />
```

Comment:
This label says Muspin where the other says Muspinsim - is that deliberate?

Reply:
Thats a typo, fixed in [6fc9c31](https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/pull/55/commits/6fc9c31be7d2bbb4be3d213d49ca864d15e930a0)